src/Spectre.libGenetic/MutationOperator.cpp

```diff
@@ -52,6 +61,23 @@ Individual MutationOperator::operator()(Individual &&individual)
         };
         std::transform(individual.begin(), individual.end(), individual.begin(), swap_random_bits);
     }
-    return individual;
+
+    auto fillup = 0u;
+    for (auto bit : individual)
+    {
+        if (bit)
+        {
+            ++fillup;
+        }
+    }
```

Comment:
Same optimization as mentioned previously can be applied here.

Reply:
Done